app/controllers/cloud_volume_controller.rb

```diff
@@ -24,23 +24,41 @@ def self.display_methods
     'cloud_volume_snapshot_create' => [:snapshot_create, 'snapshot_new'],
     'cloud_volume_backup_create'   => [:backup_create,   'backup_new'],
     'cloud_volume_backup_restore'  => [:backup_restore,  'backup_select'],
+    'cloud_volume_refresh'         => [nil, 'cloud_volume_refresh'],
   }.freeze
 
   def specific_buttons(pressed)
     return false unless BUTTON_TO_ACTION_MAPPING.include?(pressed)
 
-    validate_action, ui_action = BUTTON_TO_ACTION_MAPPING[pressed]
-    if validate_action
-      validate_results = validate_item_supports_action_button(validate_action, CloudVolume)
-      if validate_results[:action_supported]
-        javascript_redirect(:action => ui_action, :id => checked_item_id)
+    if pressed == "cloud_volume_refresh"
```

Comment:
NOTE I don't know yet if this is common for other controllers but first glance I worry about special casing this